noxfile.py

```diff
@@ -175,11 +175,25 @@ def fragment_alternative_templates(session):
     fragment(session, use_ads_templates=True)
 
 
+def _add_python_settings(tmp_dir, python_settings):
+    return f"""
```

Comment:
Could you add a comment describing the purpose of this? This is adding the Python settngs to the Showcase service YAML?

Reply:
addressed.